include/swift/Serialization/ModuleFormat.h

```diff
@@ -1089,8 +1089,9 @@ namespace decls_block {
   template <unsigned Code>
   using UnaryOperatorLayout = BCRecordLayout<
     Code, // ID field
-    IdentifierIDField, // name
-    DeclContextIDField // context decl
+    IdentifierIDField,  // name
+    DeclContextIDField, // context decl
+    TypeIDField         // protocol
```

Comment:
This should really be serialized as a Decl, not a Type, no?

Reply:
Yes that makes sense given that I am only using the type to get to the decl at this point. An earlier version of this added constructors for OperatorDecls that took TypeLocs.